src/ui/Home.fs

```diff
@@ -3,10 +3,103 @@
 open Feliz
 open type Feliz.Html
 open type Feliz.prop
+open type NextUI
+open Fable.Core.JsInterop
```

Comment:
These should be sorted with an empty line between the opens and the open types.

Reply:
Sorted.